diff --git a/crypto/perlasm/x86nasm.pl b/crypto/perlasm/x86nasm.pl
index 4b664a870b..2eb1b247c6 100644
--- a/crypto/perlasm/x86nasm.pl
+++ b/crypto/perlasm/x86nasm.pl
@@ -90,7 +90,9 @@ sub ::file
 %ifidn __OUTPUT_FORMAT__,obj
 section	code	use32 class=code align=64
 %elifidn __OUTPUT_FORMAT__,win32
+%ifndef __YASM_VER__
 \$\@feat.00 equ 1
+%endif
 section	.text	code align=64
 %else
 section	.text	code
